.github/workflows/jit.yml

```diff
@@ -137,6 +137,7 @@ jobs:
     name: Free-Threaded (Debug)
     needs: interpreter
     runs-on: ubuntu-24.04
+    timeout-minutes: 90
```

Comment:
Note 90 minutes used here rather than 60, for consistency with the other two JIT jobs.